phoenicis-javafx/src/main/java/org/phoenicis/javafx/components/library/skin/LibrarySidebarToggleGroupSkin.java

```diff
@@ -43,10 +43,19 @@ public LibrarySidebarToggleGroupSkin(LibrarySidebarToggleGroup control) {
     protected ToggleButton convertToToggleButton(ShortcutCategoryDTO category) {
         final ToggleButton categoryButton = createSidebarToggleButton(category.getName());
         // TODO: store category ID in shortcut.info?
```

Comment:
@plata does it make sense to include the content of this TODO in this PR as well?

Reply:
I think it will get too huge.